core-rust/core-api-server/src/core_api/conversions/substate.rs

```diff
@@ -220,6 +227,14 @@ pub fn to_api_component_info_substate(
     })
 }
 
+pub fn to_api_package_type_info_substate(
+    _context: &MappingContext,
+    _substate: &TypeInfoSubstate,
+) -> Result<models::Substate, MappingError> {
+    // TODO: Expand this
```

Comment:
can it be addressed in this PR? or is it blocked by something?

Reply:
I am hesitant to put too much more work on this part as this substate is most likely going to be refactored/removed soon and is not immediately useful to the end user. Will expand on the comment though to be more clear.